index.js

```diff
@@ -29,23 +32,17 @@ module.exports = (api, opts) => {
       })
   })
 
-  //Add bootstrap's variables globally
-      const bootstrapVueVarImports = [
-        '@import "~bootstrap/scss/_functions.scss"',
-        '@import "~@/assets/scss/vendors/bootstrap-vue/_custom.scss"',
-        '@import "~bootstrap/scss/_variables.scss"',
-        '@import "~bootstrap/scss/_mixins.scss"',
-        '@import "~bootstrap-vue/src/_variables.scss"',
-      ]
-
-      //add custom variables
-      opts.css.loaderOptions = {
+  //Add bootstrap's variables/functions/mixins globally
+  if(opts.useScss && opts.injectAbstracts){
+    merge(opts.css, {
+      loaderOptions:{
         sass: {
-          additionalData: bootstrapVueVarImports.join('\n')
+          additionalData: bootstrapCssAbstractsImports.join('\n')
```

Comment:
I don't think we need to introduce a lodash dependency for this. Can't we just use a typical array join?

Reply:
lodash is used just for the merge function because Object.assign() is not "deep".
In the line selected is used Array.join native method

Reply:
If you don't like to use lodash.merge, I can write a utility function that reproduces the merge's behaviour...

Reply:
Apologies, I focused in on the wrong line. Your point about the deep merge makes sense, and probably not worth re-inventing the wheel.  Considering the amount of npm packages already pulled in by the vue cli and associated libraries, I don't think we need to worry too much about lodash. Will push this through.